cmd/soroban-cli/src/commands/contract/optimize.rs

```diff
@@ -18,11 +19,18 @@ pub struct Cmd {
 pub enum Error {
     #[error(transparent)]
     Wasm(#[from] wasm::Error),
+    #[cfg(feature = "opt")]
     #[error("optimization error: {0}")]
     OptimizationError(OptimizationError),
 }
 
 impl Cmd {
+    #[cfg(not(feature = "opt"))]
+    pub fn run(&self) -> Result<(), Error> {
+        todo!("Must install with \"opt\" feature");
```

Comment:
Can we return an error here? The `todo!` macro doesn't feel very appropriate, because there's not really anything for us to implement.